persistence/src/main/java/de/cyface/persistence/DefaultLocationCleaningStrategy.java

```diff
@@ -91,19 +88,17 @@ public boolean isClean(@NonNull final GeoLocation location) {
 
         // Ignore very inaccurate locations (e.g. indoors)
         if (location.getAccuracy() >= UPPER_ACCURACY_THRESHOLD) {
-            Log.d(TAG, "GeoLocation with bad accuracy filtered: " + location.getAccuracy());
             return false;
         }
 
         // Ignore locations while standing still
         if (location.getSpeed() < LOWER_SPEED_THRESHOLD) {
-            Log.d(TAG, "GeoLocation with low speed filtered: " + location.getSpeed());
             return false;
         }
 
         // Ignore locations which are too far away from their previous location (upper speed limit)
+        // noinspection RedundantIfStatement - for readability
         if (location.getSpeed() > UPPER_SPEED_THRESHOLD) {
-            Log.d(TAG, "GeoLocation with high speed filtered: " + location.getSpeed());
             return false;
         }
 
```

Comment:
You could just do `return location.getAccuracy() < UPPER_ACCURACY_THRESHOLD && location.getSpeed() > LOWER_SPEED_THRESHOLD && location.getSpeed() < UPPER_SPEED_THRESHOLD`
There actually is no need to be so verbose.